app/models/attachment.rb

```diff
@@ -7,7 +7,7 @@ class Attachment
   field :file_id, type: String
   field :file_url, type: String
 
-  embedded_in :specialist_document_edition
+  embedded_in :section_edition
```

Comment:
Yep, that's correct. I've addressed this comment in an improved commit message on b0b409ad6.